### PR TITLE
Various fixes and enhancements

### PR DESCRIPTION
**Pull request by gavin-scott**
- Pass :reboot_job_type through from boot_rfs_iso_image to set_boot_order

- Remove unused NetworkConfiguration#is_blade?

- Implement WsMan #power_on and #power_off as instance methods

- Return :on or :off from #power_state rather than a string

### CODE REVIEW

**Comment by gavin-scott** [comment]
@ripienaar with this and https://github.com/dell-asm/dell-asm-util/pull/21 ASM::WsMan and ASM::Ipmi should consistently implement power_state, power_on, power_off and reboot.

File: lib/asm/wsman.rb [hunk 1]
```diff
@@ -168,13 +169,16 @@ def boot_source_settings
       client.enumerate("http://schemas.dell.com/wbem/wscim/1/cim-schema/2/DCIM_BootSourceSetting?__cimnamespace=root/dcim")
     end
 
-    # Get power state information
+    # Get server power state
     #
-    # @return [String] The value will be "2" if the server is on and "13" if it is off.
+    # @return [Symbol] :on or :off
```
Comment (ripienaar) [inline lib/asm/wsman.rb]: you could also do ```@return [:on,:off]``` :)

**Comment by ripienaar** [comment]
love it

File: lib/asm/wsman.rb (hunk 1 above)
Comment (gavin-scott) [inline lib/asm/wsman.rb]: Ah, thanks, good to know.

**Pull request merged**